docs/geedocs/5.2.5/answer/176738.html

```diff
@@ -273,6 +273,14 @@ <h2 id="cache_configuration">Cache configuration</h2>
 <br><br>
 # the size of the allocation blocks<br>
 BLOCK_SIZE=&lt;value&gt;
+<br><br>
```

Comment:
It just occurred to me that you don't need these changes here. The PURGE and PURGE_LEVEL options weren't available in 5.2.5.